src/main/java/com/google/codeu/servlets/MapsServlet.java

```diff
@@ -0,0 +1,73 @@
+package com.google.codeu.servlets;
+
+import com.google.gson.Gson;
+import com.google.gson.JsonArray;
+import java.io.IOException;
+import java.util.Scanner;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+
+/**
+ * Returns Starbucks Locations a JSON array, e.g. [{"lat": 38.4404675, "lng": -122.7144313}]
```

Comment:
You didn't change the comment here. This class doesn't return a starbucks location. Certain functions do but not the entire class. Suggestion would be:

`/** A servelet that handles parsing JSON Starbucks data and serving in an HttpServletResponse. */`